docs/_plugins/importer_metadata.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require "jekyll-import"
+require "mercenary"
+
+module JekyllImport
+  def self.require_with_fallback(gems)
+    Array(gems).flatten
+  end
+
+  class Bridge
+    # name     : Importer command name.
+    # importer : `JekyllImport::Importer` subclass.
+    # docs     : Array[`Jekyll::Document`].
+    def initialize(name, importer, docs)
+      @name     = name
+      @cmd      = Mercenary::Command.new(name)
+      @importer = importer
+      @doc      = docs.find { |d| d.basename_without_ext == name }
+
+      importer.specify_options(@cmd)
+    end
+
+    def inject_metadata!
+      @doc.data["cmd_name"] = name
+      @doc.data["cmd_deps"] = dependencies unless dependencies.empty?
+      @doc.data["cmd_opts"] = options      unless options.empty?
+    end
+
+    private
+
+    attr_reader :name
+
+    # Gems assumed to be available either via Ruby Stdlib or prior installation.
+    STDLIB = %w(csv date fileutils json net/http open-uri rss rubygems time uri yaml jekyll).freeze
+    private_constant :STDLIB
+
+    def dependencies
+      @dependencies ||= begin
+        deps = @importer.require_deps - STDLIB
+        deps.map! { |dep| dep.start_with?("active_support") ? "activesupport" : dep.split("/")[0] }
+        deps.uniq!
+        deps.sort!
+        deps
+      end
+    end
+
+    def options
+      @options ||= begin
+        @cmd.options.map do |o|
+          hsh = { "switch" => o.long }
+          if %r!(?<desc>.+?)(?:\z| \(default: (?<default_value>.*)\))! =~ o.description
```

Comment:
Would be a good idea to add `o.default_value` or something to Mercenary, then we wouldn't need to rely on this. https://github.com/jekyll/mercenary/blob/master/lib/mercenary/option.rb

Reply:
This is indeed a good idea. But is Mercenary still actively maintained?

Reply:
It's in the Jekyll org so we can modify it to include this! It hasn't gotten love recently since it did everything we wanted, but we could take some time to improve it. 😄